# HG changeset patch
# User adf88@interia.pl
# Date 1482658036 -3600
#      Sun Dec 25 10:27:16 2016 +0100
# Branch trunk
# Node ID 2423ccf452a8b4816fabe8a63134b818b79b55b8
# Parent  f14ce943829b0182a274038e5dec317a08b8d98e
[mq]: comment-font-height-oversize

diff -r f14ce943829b -r 2423ccf452a8 src/gfx_func.h
--- a/src/gfx_func.h	Sun Dec 25 10:00:06 2016 +0100
+++ b/src/gfx_func.h	Sun Dec 25 10:27:16 2016 +0100
@@ -161,16 +161,16 @@
 
 int GetCharacterHeight(FontSize size);
 
-/** Height of characters in the small (#FS_SMALL) font. */
+/** Height of characters in the small (#FS_SMALL) font. @note Some characters may be oversized. */
 #define FONT_HEIGHT_SMALL  (GetCharacterHeight(FS_SMALL))
 
-/** Height of characters in the normal (#FS_NORMAL) font. */
+/** Height of characters in the normal (#FS_NORMAL) font. @note Some characters may be oversized. */
 #define FONT_HEIGHT_NORMAL (GetCharacterHeight(FS_NORMAL))
 
-/** Height of characters in the large (#FS_LARGE) font. */
+/** Height of characters in the large (#FS_LARGE) font. @note Some characters may be oversized. */
 #define FONT_HEIGHT_LARGE  (GetCharacterHeight(FS_LARGE))
 
-/** Height of characters in the large (#FS_MONO) font. */
+/** Height of characters in the large (#FS_MONO) font. @note Some characters may be oversized. */
 #define FONT_HEIGHT_MONO  (GetCharacterHeight(FS_MONO))
 
 extern DrawPixelInfo *_cur_dpi;
